server_admin/topics/clients/saml/proc-entity-descriptors.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-entity-descriptors_{context}"]
+
+==== Using an entity descriptor to create a client
+
+An alternative to registering a SAML 2.0 client manually, you can import the client using a standard SAML Entity Descriptor XML file.
+
+There is an `Import` option on the Add Client page.
+
+.Add Client
+image:{project_images}/add-client-saml.png[]
+
+.Procedure
+. Click the `Select File` button.
+. Load the file that contains the XML entity descriptor information.  
+. Review the information to endure everything is set up correctly.
```

Comment:
We will "endure" this project!! 
Perhaps "ensure."

Reply:
Heh, that made me lol.

Reply:
Does still need to be changed to "ensure?"  I may be looking in the wrong place.